docs/index.md

```diff
@@ -103,7 +103,7 @@ In the example below, there are expandable panels that are nested within each ot
 
 <box>
 
-<panel header="{{ far_list_alt }} Exercises" no-close >
+<panel header=":far-list-alt: Exercises" no-close >
```

Comment:
Another un-rendered icon

![unrendered-icon](https://user-images.githubusercontent.com/31084833/52710558-961ea200-2fca-11e9-993e-b0ab101d8806.PNG)

Reply:
@Chng-Zhi-Xuan This should be rendered once you include the `vue-strap.min.js` built from MarkBind/vue-strap#97 in `assets`.